lambdas/build-cri-oauth-request/src/main/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandler.java

```diff
@@ -334,10 +334,8 @@ private EvidenceRequest getEvidenceRequestForF2F(List<SignedJWT> credentials)
                 requiredEvidences.stream()
                         .filter(
                                 requiredScores ->
-                                        requiredScores.getEvidences().size() == 1
-                                                & requiredScores.getActivity() == 0
-                                                & requiredScores.getFraud() == 0
-                                                & requiredScores.getVerification() <= 3)
+                                        requiredScores.getActivity() == 0
+                                                & requiredScores.getFraud() == 0)
```

Comment:
The lines you've removed were intentional - if you require multiple pieces of evidence or verification score >3, then you can't achieve a profile with it.

I do think we should change the evidences size to `<= 1` rather than `== 1` though - it's still ok if you don't need any evidence (but still need the verification score).

Reply:
Do you know which of these cases was causing the issue? I'm guessing it must have been the first one?

Reply:
Ok will try out with evidences <= 1 and verification <= 3.   in dev env